build_stage1.bash

```diff
@@ -4,25 +4,35 @@ mkdir -p stage1 || (echo "Could not create stage1 directory"; exit 1)
 cd stage1
 
 echo "== Configure Build"
-echo "== Build with clang-7 and lld-7"
+echo "== Build with clang and lld host compiler"
 
-CC=clang-7 CXX=clang++-7 LD=lld-7 \
-cmake 	-G Ninja \
+CC=clang CXX=clang++ LD=lld \
+	cmake -G Ninja \
 	-DBUILD_SHARED_LIBS=OFF \
 	-DCMAKE_BUILD_TYPE=Release \
 	-DCMAKE_INSTALL_PREFIX="$(pwd)/install" \
 	-DCLANG_ENABLE_ARCMT=OFF \
 	-DCLANG_ENABLE_STATIC_ANALYZER=OFF \
-	-DCLANG_VENDOR="LogMeIn" \
-	-DLLVM_ENABLE_LLD=ON \
+	-DCLANG_PLUGIN_SUPPORT=OFF \
+	-DLLVM_ENABLE_BINDINGS=OFF \
+	-DLLVM_ENABLE_OCAMLDOC=OFF \
+	-DLLVM_INCLUDE_EXAMPLES=OFF \
+	-DLLVM_INCLUDE_TESTS=OFF \
+	-DLLVM_INCLUDE_DOCS=OFF \
+	-DCOMPILER_RT_BUILD_SANITIZERS=OFF \
+	-DCOMPILER_RT_BUILD_XRAY=OFF \
+	-DCOMPILER_RT_BUILD_LIBFUZZER=OFF  \
+	-DCLANG_VENDOR="Clang-BOLT" \
+	-DCMAKE_C_FLAGS="-march=native -O3" \
+	-DCMAKE_CXX_FLAGS="-march=native -O3" \
 	-DLLVM_ENABLE_PROJECTS="clang;lld;compiler-rt" \
```

Comment:
should polly be added? i think stage1 should optimize the PGO build well, so its better to have it.